src/pages/_playground/api-explorer.tsx

```diff
@@ -108,7 +116,7 @@ export default function ApiExplorer() {
   }
 
   const handleIncludesChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
-    const includes = e.target.value.split('\n').filter((line) => line.trim())
+    const includes = e.target.value.split('\n')
```

Comment:
This made it so we couldn't add new lines to the input manually because they'd be filtered out immediately. I instead filter out blank lines when we go to _use_ the includes in the `params` during the `executeQuery` callback. Neglecting to do so results in a failed request, alas.